tools/dogfooding/dogfood.py

```diff
@@ -125,7 +126,10 @@ def dogfood(dry_run: bool, repository_url: str, repository_name: str, repository
                 dry_run=dry_run,
                 repository_url='git@github.com:DataDog/datadog-ios.git',
                 repository_name='datadog-ios',
-                repository_package_resolved_path='Datadog.xcworkspace/xcshareddata/swiftpm/Package.resolved'
+                repository_package_resolved_paths=[
+                    '.package.resolved',
+                    'DatadogApp.xcworkspace/xcshareddata/swiftpm/Package.resolved'
```

Comment:
💡 Two _package resolved_ seem redundant (they are equal), but solving this is beyond the scope of this `RUMM-1467`.